packages/api/prisma/schema.prisma

```diff
@@ -686,3 +689,16 @@ model Setting {
 
   @@map("settings")
 }
+
+model ArticlePeerInformation {
+  id         String   @id @default(cuid())
+  createdAt  DateTime @default(now())
+  modifiedAt DateTime @updatedAt
+
+  peerId String
+  peer   Peer   @relation(fields: [peerId], references: [id])
+
+  producerArticleId String
+  consumerArticleId String  @unique
+  consumerArticle   Article @relation(fields: [consumerArticleId], references: [id])
+}
```

Comment:
Missing `@@map('articles.peer-information')` to set the table name. Else the table name is `ArticlePeerInformation`